dbt2looker/generator.py

```diff
@@ -404,17 +404,40 @@ def lookml_measure(measure_name: str, column: models.DbtModelColumn, measure: mo
         m['hidden'] = measure.hidden.value
     return m
 
+def lookml_set_from_field_list(set_name, fields):
+    # spaces and linebreaks are added for proper indentation in the view.lkml file
+    fields_string = '\n      , '.join(fields)
+    return set_name + ' {\n    fields: [\n      '+  fields_string + '\n    ]\n  }'
+
+def lookml_set_of_dimensions(dimensions, dimension_groups):
+    dimension_group_fields = [
+        dg['name'] + '_' + dg['timeframes'][0]
+        for dg in dimension_groups
+    ]
+    dimension_fields = [
+        d['name']
+        for d in dimensions
+    ]
+    return dimension_group_fields + dimension_fields
```

Comment:
```suggestion
    return {"name": "details", "fields": dimension_group_fields + dimension_fields}
```

The 'lkml' package parses the value of the 'name' key and uses it as looker expects:
```
set: details {
  fields: []
}
```

Reply:
good idea, thank you.